lib/fake_data_loader.rb

```diff
@@ -61,8 +61,19 @@ def self.load_sessions
         create(
           :feedback_session,
           provider: user,
-          receiver: User.where.not(id: user.id).sample
-        )
+          receiver: User.where.not(id: user.id).sample,
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.